client/landing/stepper/hooks/use-countries.ts

```diff
@@ -4,11 +4,15 @@ import wpcom from 'calypso/lib/wp';
 type WooCountries = Record< string, string >;
 
 export function useCountries(
+	localeSlug: string | null = 'en',
 	queryOptions: Omit< UseQueryOptions< any, Error, WooCountries >, 'queryKey' > = {}
 ): UseQueryResult< WooCountries > {
 	return useQuery< any, Error, WooCountries >( {
-		queryKey: [ 'countries' ],
-		queryFn: () => wpcom.req.get( '/woocommerce/countries/regions/', { apiNamespace: 'wpcom/v2' } ),
+		queryKey: [ 'countries', localeSlug ],
+		queryFn: () =>
+			wpcom.req.get( `/woocommerce/countries/regions/?_locale=${ localeSlug }`, {
```

Comment:
On inspecting the network requests, I've noticed that with this change the `_locale` parameter is now being added twice. Here's the URL of the request that I'm seeing:

`https://public-api.wordpress.com/wpcom/v2/woocommerce/countries/regions/?_locale=es&_envelope=1&_locale=es`